lib/get/get-outdated-destination-content.js

```diff
@@ -2,6 +2,7 @@ import Promise from 'bluebird'
 import log from 'npmlog'
 
 const BATCH_CHAR_LIMIT = 1990
+const BATCH_SIZE_LIMIT = 100
```

Comment:
Won't that cause issues when the resulting response is to big? I thought even about introducing the `--max-allowed-limit` parameter we have in the export tool.